parm/fv3lam_rrfs.xml

```diff
@@ -2419,6 +2439,12 @@
        <scale>-2.0</scale>
        </param>
 
+       <param>
+       <shortname>SOILM_ON_DEPTH_BEL_LAND_SFC</shortname>
+       <pname>SOILM</pname>
```

Comment:
@EricJames-NOAA Please change SOILM into SOILMOI. The grib2 name 'SOILM' was deprecated in the grib2 tables.

Reply:
@WenMeng-NOAA welcome back!  I will work on finalizing the PR this week.